src/libs/ajax.js

```diff
@@ -51,11 +52,20 @@ const withCancellation = wrappedFetch => async (...args) => {
   }
 }
 
+const captureRetryFailure = (...args) => {
+  const errorAddress = _.flow(
```

Comment:
A comment explaining what this does would be nice. I don't think it's immediately obvious from the code.